butane_cli/src/main.rs

```diff
@@ -49,6 +49,10 @@ fn main() {
                         .help("Name to use for the migration"),
                 ),
         )
+        .subcommand(
+            clap::Command::new("detachmigration")
+                .about("Detach the latest migration")
```

Comment:
Could we give a little more information here (or use `after_help`)? I'm not sure the concept of detaching a migration will be intuitive, so I think we should explain what it means to do so and why one might want to do so (a summary of what's described in #91)